client/tx/search.go

```diff
@@ -184,3 +198,50 @@ func SearchTxRequestHandlerFn(cliCtx context.CLIContext, cdc *codec.Codec) http.
 		utils.PostProcessResponse(w, cdc, txs, cliCtx.Indent)
 	}
 }
+
+func parseHTTPArgs(r *http.Request) (tags []string, page, limit int, err error) {
```

Comment:
For a function this large, I would avoid the blank/empty returns, and use explicit returns.